server/svix-server/src/v1/endpoints/message.rs

```diff
@@ -217,74 +216,57 @@ async fn create_message(
     // Should never happen since you're giving it an existing Application, but just in case
     .ok_or_else(|| Error::Generic(format!("Application doesn't exist: {}", app.id)))?;
 
-    let txn = db.begin().await?;
-    let db = &txn;
-
     let msg = message::ActiveModel {
         app_id: Set(app.id.clone()),
         org_id: Set(permissions.org_id),
         ..data.into()
     };
     let msg = msg.insert(db).await?;
 
-    let trigger_type = MessageAttemptTriggerType::Scheduled; // Just laying the groundwork for when we support passing it
-    let empty_channel_set = HashSet::new();
-    let mut msg_dests = vec![];
-    let mut tasks = vec![];
-    for endp in create_message_app.endpoints
-        .into_iter()
-        .filter(|endp| {
-            // No disabled or deleted enpoints ever
-          	!endp.disabled && !endp.deleted &&
-            (
-                // Manual attempt types go throguh regardless
-                trigger_type == MessageAttemptTriggerType::Manual
-                || (
-                        // If an endpoint has event types and it matches ours, or has no event types
-                        endp
-                        .event_types_ids
-                        .as_ref()
-                        .map(|x| x.0.contains(&msg.event_type))
-                        .unwrap_or(true)
-                    &&
-                        // If an endpoint has no channels accept all messages, otherwise only if their channels overlap.
-                        // A message with no channels doesn't match an endpoint with channels.
-                        endp
-                        .channels
-                        .as_ref()
-                        .map(|x| !x.0.is_disjoint(msg.channels.as_ref().map(|x| &x.0).unwrap_or(&empty_channel_set)))
-                        .unwrap_or(true)
-                ))
-        })
-    {
-        let msg_dest = messagedestination::ActiveModel {
-            msg_id: Set(msg.id.clone()),
-            endp_id: Set(endp.id.clone()),
-            next_attempt: Set(Some(Utc::now().into())),
-            status: Set(MessageStatus::Sending),
-            ..Default::default()
-        };
-        msg_dests.push(msg_dest);
-        tasks.push(
-            MessageTask::new_task(
-                msg.id.clone(),
-                app.id.clone(),
-                endp.id, MessageAttemptTriggerType::Scheduled));
-    }
-    if !msg_dests.is_empty() {
-        messagedestination::Entity::insert_many(msg_dests)
-            .exec(db)
+    let trigger_type = MessageAttemptTriggerType::Scheduled;
+    if create_message_app
+    .endpoints
+    .iter()
+    .any(|endpoint| {
```

Comment:
Looking for others thoughts here: we are including this check to see if this message even has any endpoints to send to, but it does not affect the response type. At the cost of bloating the queue, we could just remove this check, and let potentially noop messages in the queue, which the worker will resolve as noops and `ack` without issue. This will save us the network call(s) to fetch `create_message_app` here in this handler.